tests/tests/uia_tests.cpp

```diff
@@ -519,6 +573,16 @@ BOOST_AUTO_TEST_CASE( asset_name_test )
       sign( tx_hf620, alice_private_key );
       PUSH_TX( db, tx_hf620 );
 
+      // assets with invalid characters should not be allowed
```

Comment:
Putting tests here means only checks after the hard fork. Please make sure it's appropriate.